exchanges/okcoin/okcoin_convert.go

```diff
@@ -0,0 +1,78 @@
+package okcoin
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+)
+
+type okcoinMilliSec time.Time
+
+// UnmarshalJSON deserializes timestamp information to time.Time
+func (o *okcoinMilliSec) UnmarshalJSON(data []byte) error {
+	var timeMilliSecond interface{}
+	err := json.Unmarshal(data, &timeMilliSecond)
+	if err != nil {
+		return err
+	}
+	switch value := timeMilliSecond.(type) {
+	case string:
+		if value == "" {
+			*o = okcoinMilliSec(time.Time{})
+			return nil
+		}
+		timeInteger, err := strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			return err
+		}
+		*o = okcoinMilliSec(time.UnixMilli(timeInteger))
+	case int64:
+		*o = okcoinMilliSec(time.UnixMilli(value))
+	case float64:
+		*o = okcoinMilliSec(time.UnixMilli(int64(value)))
+	case float32:
+		*o = okcoinMilliSec(time.UnixMilli(int64(value)))
+	}
+	return nil
+}
+
+// Time returns a time.Time instance from okcoinMilliSec instance
+func (o *okcoinMilliSec) Time() time.Time {
+	return time.Time(*o)
+}
+
+type okcoinNumber float64
+
+// UnmarshalJSON a custom JSON deserialization function for numeric values to okcoinNumber instance.
+func (a *okcoinNumber) UnmarshalJSON(data []byte) error {
+	var value interface{}
+	err := json.Unmarshal(data, &value)
+	if err != nil {
+		return err
+	}
+	switch val := value.(type) {
+	case string:
+		if val == "" {
+			*a = okcoinNumber(0)
```

Comment:
I added this because if we try to **_unmarshal_** a JSON data into a struct more than once and if the _last_  JSON has "" empty string in that okcoinNumber field, we have to reset the value of that okcoinNumber instance to 0.

Reply:
Oh right, if you can add a comment for that, that would be awesome. 👍